docs/installation/requirements.md

```diff
@@ -58,17 +63,19 @@ is compatible with such Xcode/iOS versions:
 | Xcode 14.3 / iOS 16.4 | 4.21.7 |
 | Xcode 15 / iOS 17 | 4.35.0 |
```

Comment:
just fyi: XCUTest driver 4.35.0+ can work with iOS 17 (as minimal Xcode 15 support). My recommendation for iOS 17 is XCUITest driver 5.3.3 (as some fixes were in the version).
Perhaps here we can say the minimal version, though.